python/ray/autoscaler/v2/instance_manager/common.py

```diff
@@ -163,6 +163,35 @@ def _record_status_transition(
             )
         )
 
+    @staticmethod
```

Comment:
Curious how does v1 handle stuck instances?

Reply:
i don't think there's explicit stuck instance w/o the transition graph - it handles failures (e.g. ray install failures), but don't monitor issues when things get stuck.